src/style.css

```diff
@@ -0,0 +1,74 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+.search {
+  display: flex;
+  align-items: center;
+  margin: 20px auto;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  width: 100%;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+.big-search {
+  height: 70px;
+}
+
+.small-search {
+  height: 42px;
+}
+
+.big-search img {
+  padding: 24px 15px 25px 25px;
+}
+
+.small-search img {
+  width: 11px;
+  height: 11px;
+  vertical-align: middle;
+  padding: 14px 9px 15px 12px;
+}
+
+input {
+  font-family: "Avenir" , sans-serif;
+  border: none;
+
+}
+
+.big-search input {
+  font-size: 16px;
+  padding: 0 0 0 3px ;
+  line-height: 22px;
+}
+
+.small-search input {
+  font-size: 14px;
+  padding: 0 0 0 1px;
```

Comment:
Do you really need this small paddings? And if you really need the, why don't just set padding-left?